run-seed-shisha.sh

```
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# シーシャシードデータ実行スクリプト
+
+echo "🚀 シーシャシードデータを実行中..."
+
+cd backend
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)